### PR TITLE
Add cross references to virtual machine docs

### PR DESCRIPTION
**Pull request by howardjohn**
To help these be more discoverable

### CODE REVIEW

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 1]
```diff
@@ -438,6 +438,14 @@ Run the following commands on the virtual machine you want to add to the Istio m
     Hello version: v1, instance: helloworld-v1-578dd69f69-fxwwk
     {{< /text >}}
 
+## Next Steps
+
+For more information about virtual machines:
+
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
```
Comment (smawson) [inline content/en/docs/setup/install/virtual-machine/index.md:445]: This one doesn't seem like a Next Step, seems like something they should be able to read about at the top? I'd move this one up into the intro to this page.

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 2]
```diff
@@ -438,6 +438,14 @@ Run the following commands on the virtual machine you want to add to the Istio m
...
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
```
Comment (smawson) [inline content/en/docs/setup/install/virtual-machine/index.md:446]: The wording of <LINK> to learn more about ... is a bit weird, do we use that elsewhere?

I would have it more like:

- LINK to troubleshoot issues with virtual machines.
- LINK to set up an example deployment of virtual machines.

**Comment by howardjohn** [comment]
Thanks @smawson , updated

**Review by smawson — APPROVED** [review]

File: content/en/docs/ops/diagnostic-tools/virtual-machines/index.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ test: n/a
 
 This page describes how to troubleshoot issues with Istio deployed to Virtual Machines.
 Before reading this, you should take the steps in [Virtual Machine Installation](/docs/setup/install/virtual-machine/).
+Additionally, understanding [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) can help understand how each component interacts.
```
Comment (smawson) [inline content/en/docs/ops/diagnostic-tools/virtual-machines/index.md:12]:
```suggestion
Additionally, [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) can help you understand how the components interact.
```

**Pull request merged**